src/NestedProportionalAreaChart/index.tsx

```diff
@@ -95,6 +95,7 @@ function NestedProportionalAreaChart({
               data={data}
               groupSpacing={groupSpacing}
               labels={() => ''} // Don't show PieChart labels
+              labelComponent={undefined} // Don't show PieChart labels
```

Comment:
Which of the two really works ?

Reply:
The second completely removes the component (necessary for VictoryTooltip)... The above was good enough when we were using VictoryLabel